crates/cast/src/starknet_commands/account/create.rs

```diff
@@ -87,15 +98,16 @@ pub async fn create(
     let mut items: serde_json::Value = serde_json::from_str(&contents)
         .map_err(|_| anyhow!("Failed to parse accounts file at {accounts_file_path}"))?;
 
-    let network_value = get_network(network)?.get_value();
+    let network_name = chain_id_to_network_name(chain_id);
+    println!("Detected chain_id: {chain_id:#x}");
 
-    if !items[network_value][&name].is_null() {
+    if !items[&network_name][&name].is_null() {
         return Err(anyhow!(
-            "Account with provided name already exists in this network"
+            "Account with provided name already exists in network {network_name}"
```

Comment:
I wonder if we should also map starknet_goerli -> testnet (other nets as well). Perhaps people are more familiar with these networks names, and not their ids?

Reply:
You mean the logic in `chain_id_to_network_name`?

```rust
    match &decoded[..] {
        "SN_GOERLI" => "alpha-goerli".into(),
        "SN_GOERLI2" => "alpha-goerli2".into(),
        "SN_MAIN" => "alpha-mainnet".into(),
        _ => decoded,
    }
```

If so, with the current approach I had in mind that it's still somewhat compatible with the original openzeppelin accounts file format. The network names are the same (`"alpha-mainnet"`, `"alpha-testnet2"`) instead of the chain_ids (`"SN_MAIN"`, `"SN_GOERLI2"`) and for custom ones it's just the decoded chain id.

Reply:
I mean, most people (I imagine) know what is `testnet`, and may not know what is `alpha-goerli`, we do not actually get `alpha-testnet2` anywhere, right? or am I missing something?

Reply:
No, only `alpha-goerli`, but this is how it is in the openzeppelin's file though.

Reply:
yup, but here you display it to the user in an error message, right? I suppose most won't be familiar with the oz file, and may not necessarily know why (since they use testnet), the error says 'account exists in alpha-goerli`. 

Maybe it's just a nitpick, and in reality this is a smaller issue than I potray it to be, but I can se this be a bit vague for some users.

Reply:
I see. I actually though about changing the message to something among the lines of `... already exists with network with chain_id {decoded_chain_id}`, where decoded_chain_id would be just the decoded hex-chain_id (so "SN_GOERLI" for example)
wdyt

Reply:
This could be a way to go, yes. This would also be a lot easier to maintain.

Reply:
But we still do want the 3 default nets to be named `alpha-...` in the file, right?

Reply:
yes, actually I think we must still support this, as there are already users with accounts deployed under this schema. We'll have to figure out a way to get rid of this sometime in the future, if there will be a need to do this.